[Apache Tomcat] Migrate to GA (#8385)
elastic/integrations: packages/apache_tomcat/_dev/build/docs/README.md

Metrics give you insight into the statistics of the Apache Tomcat. The `Metric` data streams collected by the Apache Tomcat integration are `cache`, `connection pool`, `memory`, `request`, `session` and `thread pool`, so that the user can monitor and troubleshoot the performance of the Apache Tomcat instance.

Data streams:

> - `access`: Collects information related to HTTP transactions between the client and the application server such as IP address of the client sending the request, the request method and URL path, the HTTP status code, the number of bytes returned to the client and more.

Reviewer: ```suggestion
- `access`: Collects information related to the HTTP transactions, client IP, response code and request processing time.
```

Reviewer: Please make the above change [here](https://docs.elastic.co/integrations/apache_tomcat#:~:text=logs%20related%20to-,overall%20performance,-of%20Java%20applications) as well.

Author: Updated, thanks
